docs/CONTRIBUTING.md

```diff
@@ -722,8 +722,8 @@ Btw. you can specify `ENSO_JVM_OPTS` to turn
 [debugging of the Engine runtime](debugger/README.md) on:
 
 ```bash
-$ export ENSO_JVM_OPTS=-agentlib:jdwp=transport=dt_socket,address=5005
-$ ./built-distribution/enso-project-manager-0.0.0-dev-linux-amd64/enso/bin/project-manager --no-log-masking -v
```

Comment:
The idea is: `$` means the line that you enter. Line without `$` prefix is output.

Reply:
There are dozens of other commands in that file that you're supposed to enter, and none start with $, so it's not very clear.

Reply:
There are nine lines with `$ ` in [the document](https://github.com/enso-org/enso/blame/f38033b037800bddef55257a3374aa9f571678aa/docs/CONTRIBUTING.md) according to _blame tool_ they were written by me, @4e6 and @mwu-tow.

I am not sure how others, but I'd prefer the `$ ` prefix notation for lines that _one is supposed to enter_.

Reply:
Yes, the `$` indicates a shell command. I think this notation originates from the default Linux prompt where `$` stands for the user shell, and `#` for the root shell.
I'd like to leave them because they visually separate the shell command and its result. Making it easier to parse the doc with your eyes. But I agree that the docs are inconsistent in this. For example, we can use the `>` symbol for PowerShell and `$` for Unix to make it uniform across the docs.

Reply:
My view:
* Don't use `$` in a code block with just command(s) that user is supposed to enter. The GH provides the "Copy" button; it is nice to be able just to click it and paste into the terminal.
* Use `$` if the command is not supposed to be used verbatim, also when we are mixing the user's command with the expected output.

Still, this is a pretty weak preference, I don't mind either way.